scripts/install-envoy-gen-validate-plugin.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -euo pipefail
+
+source scripts/executor-info.sh
+
+PLUGIN_VERSION=$1
+
+if ! type "protoc-gen-validate" > /dev/null 2>&1; then
+  echo "Installing protoc-gen-validate $PLUGIN_VERSION"
+  
+  go install github.com/envoyproxy/protoc-gen-validate@$PLUGIN_VERSION
```

Comment:
Ah and all other plugins are installed in `./.bin`, so we should do that here as well:
```suggestion
  GOBIN="./.bin" go install github.com/envoyproxy/protoc-gen-validate@$PLUGIN_VERSION
```